### DISCUSSION TITLE
Is there anyone to contribute to v2?

### DISCUSSION THREAD

**Discussion by Wernke96** [beyondcode/laravel-websockets#1040] [General]
Hi, I was wondering if there is anyone to help contribute to v2? I was just wondering if I could help out with development in any way possible.

**Comment by mpociot** [comment]
I'm currently working on the finishing touches to get v2 out of the door.
What would you like to add/improve/work on? :)

**Reply by Wernke96** [reply]
Let me take a couple of days and take a look around at the code and see how it works to get some ideas. I just found this project so I would love to be able to help in the future. Once I have an idea I will post it in this discussion. :)

**Reply by parallels999** [reply]
> What would you like to add/improve/work on? :)

@mpociot maybe this #950
